terraform/context.go

```diff
@@ -207,6 +207,7 @@ func (c *Context) Graph(typ GraphType, opts *ContextGraphOpts) (*Graph, error) {
 		opts = &ContextGraphOpts{Validate: true}
 	}
 
+	log.Printf("[INFO] terraform: building graph: %s", typ)
```

Comment:
Just added this while we're here since the debug log was unclear what graph was being built. The old graph builders used to do this.